internal/system/impl/sqlstore.go

```diff
@@ -180,6 +187,20 @@ func (s *SystemSQLStoreService) getMetadataImage(chainID tableland.ChainID, tabl
 	return fmt.Sprintf("%s/%d/%s", uri, chainID, tableID)
 }
 
+func (s *SystemSQLStoreService) getAnimationURL(chainID tableland.ChainID, tableID tables.TableID) string {
+	if s.animationRendererURI == "" {
+		return DefaultAnimationURL
+	}
+
+	uri := strings.TrimRight(s.animationRendererURI, "/")
+	if _, err := url.ParseRequestURI(uri); err != nil {
+		log.Warn().Str("uri", uri).Msg("metadata renderer uri could not be parsed")
+		return DefaultAnimationURL
+	}
```

Comment:
Maybe we can do this check in the ctor, and fail fast signaling that the configuration of the validator is incorrect instead of defaulting to a global var (which probably wasn't the intention if a configuration was provided).

Also, `uri` would have been precalculated so we don't need to do this parsing on every call.

Reply:
yeah. moving over there